web/web.ex

```diff
@@ -16,12 +16,6 @@ defmodule Elixirlight.Web do
   below.
   """
 
-  def model do
```

Comment:
I'm not sure I'd kill this just yet. It's up to you but I think I'd leave it. No cost to leave it and it could be confusing in the future when you move this to using a model (as you may do).